bitnami/pinniped/values.yaml

```diff
@@ -88,6 +88,25 @@ concierge:
     api: 10250
     proxy: 8443
 
+  ## @param concierge.configurationPorts.aggregatedAPIServerPort Concierge API configuration port
+  ## @param concierge.configurationPorts.impersonationProxyServerPort Concierge Proxy configuration port
+  ##
+  ## Changes to concierge.configuration overwrites this param.
+  ##
+  ## If changed, other YAML references to the default ports may also need to be updated
+  ##
+  configurationPorts:
+    aggregatedAPIServerPort: 10250
+    impersonationProxyServerPort: 8444
+
+  ## @param concierge.hostNetwork Concierge API and Proxy container hostNetwork
+  ##
+  hostNetwork: false
+
+  ## @param concierge.dnsPolicy Concierge API and Proxy container dnsPolicy
+  ##
+  dnsPolicy: ClusterFirst
```

Comment:
Can we set the defaults to `""` so that we can only add them if the user explicitly define them?

Reply:
Set empty strings as default values but kept `hostNetwork: false`.